docs/how-to-use-paperclip-with-minio.md

```diff
@@ -0,0 +1,37 @@
+# How to use Paperclip with Minio Server [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Paperclip is intended as an easy file attachment library for ActiveRecord. In this recipe we will learn how to configure  Minio as Object Storage backend for Paperclip.
```

Comment:
Please add link to Paperclip.  Also change PaperClip to Paperclip. 

Edit the line : In this recipe 'you' will learn not 'we' will learn.
Object Storage should be in lower case. The line should be "Minio as an object storage backend"